main.py

```diff
@@ -11,14 +12,52 @@ class Account:
     Credit Union Account
     """
 
-    def __init__(self, account_number, full_name):
+    def __init__(self, account_number, full_name, account_balance):
         self.__account_number = account_number
         self.full_name = full_name
+        self.account_balance = account_balance
 
     def account_summary(self):
-        return f"{self.full_name}'s account number is {self.__account_number}"
+        summery = f"{self.full_name}'s account number is {self.__account_number} and the account balance is {self.account_balance}"
+        return summery
+
+
+class Checking(Account):
+    """
+    Checking Account
+    """
+
+    def __init__(self, account_number, full_name, account_balance, checks="Checks"):
+        self.checks = checks
+        super().__init__(account_number, full_name, account_balance)
+
+    def account_summary(self):
+        summery = super().account_summary()
+        summery = f"{summery} and has {self.checks} check(s)"
+        return summery
+
+
+class Credit(Account):
+    """
+    Credit Card
+    """
+
+    def __init__(
+        self, account_number, full_name, account_balance, credit_card="Credit Card"
+    ):
+        self.credit_card = credit_card
+        super().__init__(account_number, full_name, account_balance)
+
+    def account_summary(self):
+        summery = super().account_summary()
+        summery = f"{summery} and has {self.credit_card} credit card(s)"
+        return summery
 
 
 if __name__ == "__main__":
-    member = Account("123456", "Luis Santiago")
-    print(member.account_summary())
+    Katy = Account("123456", "Katy Perry", "$100,000")
+    print(Katy.account_summary())
+    Miley = Checking("234567", "Miley Cyrus", "$500,000", 1)
```

Comment:
For our basic program, passing a string value here is "ok", but really you should try to use integer values for numbers where possible